public/data/ashishtom.json

```
@@ -0,0 +1,17 @@
+{
+    "name": "Ashish Thomas",
+    "bio": "Flutter and web developer",
+    "avatar": "https://github.com/alceil.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/alceil",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/ashishthomas6",
+        "icon": "twitter"
+      }
+    ]
+}
```

Still missing an endline.
```suggestion
}

```

---

done